Pocket.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -140,8 +140,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-nio.git",
       "state" : {
-        "revision" : "4ad2c3733845abd9ee8892a323b0fa0d80f37e34",
-        "version" : "2.47.0"
+        "revision" : "45167b8006448c79dda4b7bd604e07a034c15c49",
+        "version" : "2.48.0"
```

Comment:
have you merged in develop that fixes this?

Reply:
this is an updated version compared to develop, not sure why it's updating now

Reply:
https://github.com/Pocket/pocket-ios/blob/develop/Pocket.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

Reply:
could be that an exact version is relying on this dependency in the latest version as well